Flowchart/Views/Home/About.cshtml

```diff
@@ -1,10 +1,20 @@
 ﻿@{
     ViewData["Title"] = "About";
 }
+
 <h2>@ViewData["Title"].</h2>
 <h3>@ViewData["Message"]</h3>
 
 
-    <p>This software is designed to provide a simple and easy way to create flowcharts for various purposes.</p>
+    <p>This software is designed to provide a simple and easy way to create flowcharts for various purposes.
+    For a list of possible problems that may arise please look below</p>
```

Comment:
Probably a good idea to add a header here instead of just part of another paragraph. 

Look into viewbags and viewdata. You can modify their input inside the controller (which you should definitely do with how it looks now).